index.html

```diff
@@ -0,0 +1,9 @@
+<h1> Christopher Van Kampen </h1>
+<h2>About</h2>
+<p> I am currently a part time students attending CTD classes. Additionally, I am renovating our house along with my coursework.</p>
+<h2> Experience </h2>
+<ul><li>West Ottawa Public Schools </l1><li>OVD Insurance </li></ul>
+<h2> Connect </h2>
+<a title="" href="https://https://www.linkedin.com/in/vankampenc/" target="_blank">LinkedIn</a>
```

Comment:
Not sure if this link is correct, tried adding you on LinkedIn but the profile didn't pop up!

Reply:
Updated the link!